cli/cmd/replicaset.go

```diff
@@ -0,0 +1,177 @@
+	statusCmd.Flags().StringVar(&replicasetSslCertFile, "sslcertfile", "",
+		`path to an SSL certificate file for the URI case`)
+	statusCmd.Flags().StringVar(&replicasetSslCaFile, "sslcafile", "",
+		`path to a trusted certificate authorities (CA) file for the URI case`)
+	statusCmd.Flags().StringVar(&replicasetSslCiphers, "sslciphers", "",
+		`colon-separated (:) list of SSL cipher suites for the URI case`)
+	replicasetCmd.AddCommand(statusCmd)
+
+	return replicasetCmd
+}
+
+// internalReplicasetStatusModule is a "status" command for the replicaset module.
+func internalReplicasetStatusModule(cmdCtx *cmdcontext.CmdCtx, args []string) error {
+	orchestrator, err := getOrchestrator()
+	if err != nil {
+		return err
+	}
+
+	connectCtx := connect.ConnectCtx{
+		Username:    replicasetUser,
+		Password:    replicasetPassword,
+		SslKeyFile:  replicasetSslKeyFile,
+		SslCertFile: replicasetSslCertFile,
+		SslCaFile:   replicasetSslCaFile,
+		SslCiphers:  replicasetSslCiphers,
+	}
+
+	var (
+		connOpts          connector.ConnectOpts
+		runningCtx        running.RunningCtx
+		isInstanceConnect bool
+		isApplication     bool
+	)
+	if err := running.FillCtx(cliOpts, cmdCtx, &runningCtx, args); err == nil {
+		if len(runningCtx.Instances) == 1 {
+			if connectCtx.Username != "" || connectCtx.Password != "" {
+				err = fmt.Errorf("username and password are not supported" +
+					" with a connection via a control socket")
+				return err
+			}
+			connOpts = makeConnOpts(
+				connector.UnixNetwork,
+				runningCtx.Instances[0].ConsoleSocket,
+				connectCtx,
+			)
+			isInstanceConnect = true
+			colonIds := strings.Index(args[0], string(running.InstanceDelimiter))
+			if colonIds != -1 {
+				// Re-fill context for an application.
+				appName = args[0][:colonIds]
+				err := running.FillCtx(cliOpts, cmdCtx, &runningCtx, []string{appName})
+				if err != nil {
+					// Should not happen.
+					return err
+				}
+			}
+		}
+		isApplication = true
+	} else {
+		var err error
+		connOpts, _, err = resolveConnectOpts(cmdCtx, cliOpts, &connectCtx, args)
+		if err != nil {
+			return err
+		}
+		isInstanceConnect = true
+	}
+
+	var conn connector.Connector
+	if isInstanceConnect {
+		// Connecting to the instance.
+		var err error
+		conn, err = connector.Connect(connOpts)
+		if err != nil {
+			return fmt.Errorf("unable to establish connection: %s", err)
+		}
+		defer conn.Close()
+	}
+
+	return replicaset.Status(replicaset.StatusCtx{
+		RunningCtx:    runningCtx,
+		IsApplication: isApplication,
+		Conn:          conn,
+		Orchestrator:  orchestrator,
+	})
+}
+
+// getOrchestartor returns a chosen orchestrator or an unknown one.
+func getOrchestrator() (explorer.Orchestrator, error) {
+	orchestrator := explorer.OrchestratorUnknown
+	cnt := 0
+	for k, v := range orchestratorsEnabled {
+		if *v {
+			orchestrator = k
+			cnt++
+		}
+	}
+	if cnt > 1 {
+		return orchestrator, fmt.Errorf("only one type of orchestrator can be forced")
```

Comment:
A user could force an `orchestrator` instead of usage of an auto-detected value:

https://github.com/tarantool/tt/blob/0b8cbdd29b05c3994faf83cc31af2b6e8661f789/cli/cmd/replicaset.go#L65-L70